egui/src/widgets/slider.rs

```diff
@@ -85,9 +93,44 @@ impl<'a> Slider<'a> {
         }
     }
 
+    /// Creates a new vertical slider.
+    pub fn new_vertical<Num: emath::Numeric>(
```

Comment:
It seems unnecessary to have both this and the `.orientation` builder.

Perhaps `Slider::new(value, 0..100).vertical()` is a nicer way to construct a vertical slider anyway? And then we don't need `new_vertical`, `from_get_set_vertical` and `from_get_set_inner`